### DISCUSSION TITLE
Peak calling with nucleosomal free reads

### DISCUSSION THREAD

**Discussion by Y-ogativity** [GreenleafLab/ArchR#1872] [Questions / Documentation] [answered]
Hi,

Thank you for developing and maintaining such a valuable ATAC-seq analysis package! 
I would like to know how to call peaks with only nucleosomal free reads and identify maker peaks.

I aim to compare motif enrichment results across species. However, I observed that distinct fragment length distribution across my snATAC dataset, which I believe was caused by the difference in the sample quality. The peak calling with nucleosomal free reads would avoid such technical confounds and make a reasonable comparison of TF motif enrichment.

**Comment by rcorces** [comment, marked as answer]
this functionality is not available in ArchR. You would have to filter your fragment files yourself if you only want to focus on sub-nucleosome reads.

**Reply by Y-ogativity** [reply]
I see. Thank you for the quick reply.
